add_glyphs.py

```diff
@@ -163,6 +167,16 @@ def add_glyph_data(font, seqs, seq_to_advance, vadvance):
   hmtx = font['hmtx'].metrics
   vmtx = font['vmtx'].metrics
 
+  # Add glyf table so empty glyphs will be added to ensure compatibility
+  # with systems requiring a glyf table, like Windows 10.
+  if add_glyf:
+    pen = TTGlyphPen(None)
+    empty_glyph = pen.glyph()
+    font['loca'] = _l_o_c_a.table__l_o_c_a()
```

Comment:
you could also use the newTable function from fontTools.ttLib, which returns a new empty instance of table with given tag, so you don't need to import the table module with the long names, e.g. `newTable("loca")` or `newTable("glyf")`

Reply:
👍 Thanks for the tip!